### PR TITLE
week1 homework done

### PR DESCRIPTION
**Pull request by genegr88**
homework week1 part 2 done

### CODE REVIEW

File: Week1/homework/world-queries.txt [hunk 1]
```diff
@@ -1,40 +1,186 @@
...
+7.What is the population in Asia?
+ SELECT continent,SUM(population) FROM country WHERE continent='Asia' GROUP BY continent;
++-----------+-----------------+
+| continent | SUM(population) |
++-----------+-----------------+
+| Asia      |      3705025700 |
++-----------+-----------------+
+1 row in set (0.08 sec)
+8.Which languages are spoken in the Region of ‘South America’
+ SELECT DISTINCT language FROM countrylanguage JOIN country on countrylanguage.countrycode=country.code WHERE continent='South America';
```
Comment (pcelentano) [inline Week1/homework/world-queries.txt]: nice use of distinct 👍

File: Week1/homework/world-queries.txt [hunk 2]
```diff
@@ -1,40 +1,186 @@
...
+| English          |
+| Creole French    |
+| Hindi            |
+| Sranantonga      |
+| Goajiro          |
+| Warrau           |
++------------------+
+21 rows in set (0.01 sec)
+9.What are the languages spoken on all cities named ‘Barcelona’ (you may need to join 3 tables =) )
+ SELECT DISTINCT countrylanguage.language from countrylanguage
```
Comment (pcelentano) [inline Week1/homework/world-queries.txt]: but this use of distinct is a little more dangerous ! why did you need a distinct here.. maybe you were getting 2 times spanish ? if so.. my inner data scientist wants to know why, how about we print the city name and the country name ?

**Comment by unmeshvrije** [comment]
Please re-open the PR after you make changes
